### PR TITLE
fix: Gammabright is now not changed incorrectly when opening settings screen

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/user/GammabrightFeature.java [hunk 1]
```diff
@@ -27,26 +27,24 @@ public class GammabrightFeature extends UserFeature {
...
         if (event.getNewState() != WorldStateManager.State.WORLD) return;
-        if (!gammabrightEnabled) return;
 
-        lastGamma = McUtils.options().gamma;
-        McUtils.options().gamma = 1000d;
+        applyGammabright();
     }
 
-    private void onGammabrightKeyPress() {
+    private void applyGammabright() {
```
Comment (Incompleteusern) [inline common/src/main/java/com/wynntils/features/user/GammabrightFeature.java:39]: `applyGammabright` seems to be a toggle of `gammabrightEnabled`. As such, why are we calling it on world state change, which toggles the gamma as the world changes, and when the config is changed, which negates the change?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/user/GammabrightFeature.java:39]: Right, thanks

File: common/src/main/java/com/wynntils/features/user/GammabrightFeature.java [hunk 2]
```diff
@@ -50,6 +50,11 @@ private void applyGammabright() {
         ConfigManager.saveConfig();
     }
 
+    private void toggleGammaBright() {
+        gammabrightEnabled = !gammabrightEnabled;
```
Comment (Incompleteusern) [inline common/src/main/java/com/wynntils/features/user/GammabrightFeature.java:51]: You seem to have forgotten the toggling still present within applyGammabright itself

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/user/GammabrightFeature.java:51]: ape mode

**Review by Incompleteusern — APPROVED** [review]

**Pull request merged**